src/main/resources/static/css/home-page-style.css

```
@@ -0,0 +1,81 @@
+body{
+    background-color: #42BFDD;
+
+}
+
+p{
+    font-size: 24px;
+  }
+
+h2{
+    font-size: 48px;
+  }
+
+.button {
+    background-color: #1D77C3; 
+    border: none;
+    color: white;
+    padding: 15px 50px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    width: 85%;
+    gap: min(2px, 1%);
+  
+  }
+  .button:hover {
```

Indenting, and gap between rules
```suggestion
}

.button:hover {
```